Index: gnome-settings-daemon/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
===================================================================
--- gnome-settings-daemon.orig/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
+++ gnome-settings-daemon/data/org.gnome.settings-daemon.peripherals.gschema.xml.in
@@ -112,7 +112,7 @@
       <description>Select the touchpad scroll method. Supported values are: “disabled”, “edge-scrolling”, “two-finger-scrolling”.</description>
     </key>
     <key name="tap-to-click" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Enable mouse clicks with touchpad</summary>
       <description>Set this to TRUE to be able to send mouse clicks by tapping on the touchpad.</description>
     </key>
