Index: gnome-settings-daemon/data/org.gnome.settings-daemon.plugins.color.gschema.xml.in
===================================================================
--- gnome-settings-daemon.orig/data/org.gnome.settings-daemon.plugins.color.gschema.xml.in
+++ gnome-settings-daemon/data/org.gnome.settings-daemon.plugins.color.gschema.xml.in
@@ -22,17 +22,17 @@
       <description>This temperature in Kelvin is used to modify the screen tones when night light mode is enabled. Higher values are bluer, lower redder.</description>
     </key>
     <key name="night-light-schedule-automatic" type="b">
-      <default>true</default>
+      <default>false</default>
       <summary>Use the sunrise and sunset</summary>
       <description>Calculate the sunrise and sunset times automatically, from the current location.</description>
     </key>
     <key name="night-light-schedule-from" type="d">
-      <default>20.00</default>
+      <default>00.00</default>
       <summary>The start time</summary>
       <description>When “night-light-schedule-automatic” is disabled, use this start time in hours from midnight.</description>
     </key>
     <key name="night-light-schedule-to" type="d">
-      <default>6.00</default>
+      <default>00.00</default>
       <summary>The end time</summary>
       <description>When “night-light-schedule-automatic” is disabled, use this end time in hours from midnight.</description>
     </key>
